samples/Patient and Population Services G10/docs/deployment.md

```diff
@@ -50,7 +50,7 @@ Next you will need to clone this repository and prepare your environment for dep
     azd up
     ```
 
-*NOTE:* This will take about an hour to deploy, mainly for Azure API Management. You can continue with Azure Active Directory setup below.
+*NOTE:* This will take about an 15 minutes to deploy.
```

Comment:
Minor grammar. Maybe: 'This will take around 15 minutes to deploy".

Reply:
Fixed! Thanks for that catch!